App.js

```diff
@@ -6,13 +6,30 @@
  * @flow
  */
 
-import React, {Component} from 'react';
-import Home from './src/scenes/home/Home'
+import React from 'react';
+import { createAppContainer, createStackNavigator } from 'react-navigation';
 
-export default class App extends Component {
-  render() {
-    return (
-      <Home />
-    );
-  }
+import Details from './src/scenes/details/Details';
+import Home from './src/scenes/home/Home';
+
+export default function App() {
+  return (
+    <AppContainer />
+  );
 }
+
+const AppNavigator = createStackNavigator(
+  {
+    Home: {
+      screen: Home,
+    },
+    Details: {
+      screen: Details,
+    },
+  },
+  {
+    initialRouteName: 'Home',
```

Comment:
What do you think about creating a Route file with an enum? In the enum you can define all your routes.

_**export enum Route {**_
  _**Home = 'Home',**_
  _**Details = 'Details ',**_
_**}**_

Then you can use it like this:

_**import { Route } from ...**_

_**initialRouteName: Route.Home,**_